benchmarks/allocator_no-op_benchmarks.cpp

```diff
@@ -0,0 +1,68 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include "benchmark/benchmark.h"
+#include "umpire/config.hpp"
+#include "umpire/Allocator.hpp"
+#include "umpire/ResourceManager.hpp"
+
+static const int RangeLow{4};
+static const int RangeHi{1024};
+
+class NoOpAllocatorBenchmark : public benchmark::Fixture
+{
+public:
+  using ::benchmark::Fixture::SetUp;
+  using ::benchmark::Fixture::TearDown;
+
+  NoOpAllocatorBenchmark() {}
+
+  void SetUp(benchmark::State&) {
+    auto& rm = umpire::ResourceManager::getInstance();
+    alloc = rm.getAllocator("NO_OP");
+  }
+ 
+  void TearDown(benchmark::State&) {
+    alloc.getAllocationStrategy()->release();
```

Comment:
This line is no longer needed since you are dealing directly with the resource allocator.  `release()` is only meaningful for pools.

Reply:
Yea I thought you might say that lol it feels weird to not have to release memory (but yea, I'll take that out, thanks)